### PR TITLE
feat: provide warning on connection with unverified users [SQSERVICES-1679]

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1679" title="SQSERVICES-1679" target="_blank">StorySQSERVICES-1679</a>  [Web] Unverified user warning on connection request and in-chat
  </td></table>
  <br />
 

___
### Sub Tickets
[SQSERVICES-1853] [SQSERVICES-1854] [SQSERVICES-1855]

### Feature

- Add a warning on contact request to make sure unverified users are who they claim

image

### Bug Fix

- Links to the help center were obsolete and until now were redirected by the wire website one by one, and only in English
- Modified the links in `externalRoute.ts` to point directly to the help center, and allow localization

[SQSERVICES-1853]: https://wearezeta.atlassian.net/browse/SQSERVICES-1853?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SQSERVICES-1854]: https://wearezeta.atlassian.net/browse/SQSERVICES-1854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SQSERVICES-1855]: https://wearezeta.atlassian.net/browse/SQSERVICES-1855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/script/externalRoute.ts [hunk 1]
```diff
@@ -62,6 +64,13 @@ export const getWebsiteUrl = (path: string = '', pkCampaign?: string): string |
   }
   return undefined;
 };
+export const getHelpCenterUrl = (path: string = ''): string | undefined => {
```
Comment (PatrykBuniX) [inline src/script/externalRoute.ts]: Would be nice to be more strict about types here, eg. add `as const` to `URL_PATH` and use type `typeof URL_PATH[keyof typeof URL_PATH]`

**Review by przemvs — APPROVED** [review]

File: src/script/externalRoute.ts (hunk 1 above)
Comment (V-Gira) [reply src/script/externalRoute.ts]: good idea, I need to clean up the rest of the file anyway, I think there's a whole bunch of unused links

File: src/script/components/Modals/UserModal/UserModal.tsx [hunk 2]
```diff
@@ -98,6 +101,29 @@ const UserModalUserActionsSection: React.FC<UserModalUserActionsSectionProps> =
...
+      <Icon.Info css={{height: '1rem', margin: '0.2em 1em', minWidth: '1rem'}} />
+      <p css={{fontSize: 'var(--font-size-medium)'}}>
+        {t('userNotVerified', {user: user.name()})}
+        <Link
+          css={{fontSize: 'var(--font-size-medium)', margin: '0 1em'}}
+          variant={LinkVariant.PRIMARY}
+          targetBlank
+          href={getPrivacyUnverifiedUsersUrl()}
+        >
+          Learn more
```
Comment (atomrc) [inline src/script/components/Modals/UserModal/UserModal.tsx]: hard coded string spotted 👀

File: src/script/externalRoute.ts [hunk 3]
```diff
@@ -78,8 +87,10 @@ export const getManageTeamUrl = (utmSource?: string): string => getTeamSettingsU
 
 export const getCreateTeamUrl = (): string =>
   Config.getConfig().FEATURE.ENABLE_ACCOUNT_REGISTRATION && `${Config.getConfig().URL.TEAMS_BASE}/register/email`;
+export const getDecryptError = (): string => getHelpCenterUrl(URL_PATH.DECRYPT_ERROR_1);
```
Comment (atomrc) [inline src/script/externalRoute.ts]:
```suggestion
export const getDecryptErrorUrl = (): string => getHelpCenterUrl(URL_PATH.DECRYPT_ERROR_1);
```

Let's make it consistent with the other urls

File: src/script/externalRoute.ts [hunk 4]
```diff
@@ -90,3 +101,15 @@ export const addLocaleToUrl = (url?: string): string => {
   const websiteLanguage = language == 'de' ? language : 'en';
   return url.replace(Config.getConfig().URL.WEBSITE_BASE, `${Config.getConfig().URL.WEBSITE_BASE}/${websiteLanguage}`);
 };
+
+export const addLocaleToHelpCenterUrl = (url?: string): string => {
```
Comment (atomrc) [inline src/script/externalRoute.ts]:
```suggestion
const addLocaleToHelpCenterUrl = (url?: string): string => {
```

Let's keep it private for now

**Review by atomrc — CHANGES_REQUESTED** [review]
Looks good overall :) Just a few things

**Review by atomrc — APPROVED** [review]

File: src/script/components/MessagesList/Message/MemberMessage.tsx [hunk 5]
```diff
@@ -176,6 +176,16 @@ const MemberMessage: React.FC<MemberMessageProps> = ({
...
+      <div className="message-header">
+        <div className="message-header-icon" />
+        <p className="message-header-label">{t('conversationNewConversation')}</p>
+      </div>
+      <div className="message-header">
+        <div className="message-header-icon message-header-icon--svg text-foreground">
+          <Icon.Info />
+        </div>
+        <p className="message-header-label">{t('conversationUnverifiedUserWarning')}</p>
+      </div>
```
Comment (V-Gira) [inline src/script/components/MessagesList/Message/MemberMessage.tsx]: refactor of this 
image

**Pull request merged**